rules/opentitan.bzl

```diff
@@ -726,12 +771,107 @@ pick_correct_archive_for_device = rv_rule(
     toolchains = ["@rules_cc//cc:toolchain_type"],
 )
 
+def opentitan_multislot_flash_binary(
+        name,
+        srcs,
+        image_size,
+        key_per_binary,
+        offset_per_binary,
+        platform = OPENTITAN_PLATFORM):
+    """A helper macro for generating multislot OpenTitan binary flash images.
+
+    This macro is mostly a wrapper around the `assemble_flash_image` rule, that
+    invokes `opentitantool` to stitch together multiple `opentitan_flash_binary`
+    images to create a single image for bootstrapping. This enables efficient
+    testing by only requiring one boostrap operation to load both silicon
+    creator and owner stages of flash.
```

Comment:
Not just for efficiency: we cannot do this any other way since bootstrap erases the flash from programming.

Reply:
Thanks for the explanation, I updated the comment to clarify.